app/admin_app.py

```diff
@@ -0,0 +1,83 @@
+from flask import Flask
+from flask import jsonify
+from flask import make_response
+from flask import render_template
+from flask import request
+import json
+
+app = Flask(__name__)
+# set up freezer config
+app.config.from_pyfile('settings.py')
+# read json data
+data = json.loads(open('data.json', 'r').read())
+
+@app.route('/', methods=['GET', 'POST'])
+def index_page():
+    if request.method == 'GET':
+		return render_template('form.html')
+    else:
+        val = request.form.get('button')
+        if val == 'apis':
+            return apis(request)
+        elif val == 'challenges':
+            return challenges(request)
+        else:
+			return render_template('form.html')
+
+def apis(request):
+	api_info = {
+		'name':request.form.get('company_name'),
+		'image': request.form.get('image'),
+		'description': request.form.get('description'),
+		'link': request.form.get('link'),
+		'keywords': request.form.get('search_keywords').split(','),
```

Comment:
`[keyword.strip() for keyword in request.form.get('search_keywords').split(',')]` so we can do split on comma-space as well as comma

Reply:
Will do